index.html

```diff
@@ -25,7 +25,7 @@
               <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
               <ul class="overlay-content">
                  <li><a   href="javascript:void(0);" onclick="menuOptions('#about')" >Portfolio<i class="fa fa-angle-right"></i></a></li>
-                  <li><a href="javascript:void(0);" onclick="menuOptions('#contact-info')">About<i class="fa fa-angle-right"></i></a></li>
+                  <li><a href="javascript:void(0);" onclick="menuOptions('#about-1')">About<i class="fa fa-angle-right"></i></a></li>
```

Comment:
- [ ] [OPTIONAL] You are using an event listener inside HTML file, please consider using a separate javascript file for scripts